doc/about.rst

```diff
@@ -1,8 +1,11 @@
-About Panel
-===========
+About
+=====
 
-Panel is completely open source, available under a BSD license freely for both commercial and non-commercial use. Panel is part of the HoloViz ecosystem and works well with all the HoloViz tools.
+Panel is completely open source, available under a BSD license freely for both commercial and non-commercial use.
 
-Panel was originally developed with the support of `Anaconda Inc. <https://anaconda.com>`_, and is now maintained by Anaconda developers and community contributors.
+Panel is part of the `HoloViz <https://holoviz.org>`_ family of tools. The `holoviz.org <https://holoviz.org>`_ website shows how to use Panel together with other libraries to solve complex problems, with detailed tutorials and examples. Example projects using Panel are at
```

Comment:
```suggestion
Panel is part of the `HoloViz <https://holoviz.org>`_ family of tools. The `holoviz.org <https://holoviz.org>`_ website shows how to use Panel together with other libraries to solve complex problems, with detailed tutorials and examples. You can see a variety of projects using Panel at
```

(The original wording saying that "examples projects are at" seems a bit blunt and also doesn't acknowledge that there can be many Panel projects all over the place; these are just some useful ones to check out.)